### PR TITLE
Add MIT License

### PR DESCRIPTION
**Pull request by ossanna16**
Explicitly adding the MIT license to allow people to choose which license terms they use for code examples and snippets shared with-in the discussions area of this project and organization.

### CODE REVIEW

File: LICENSE-CODE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 GitHub Community
```
Comment (martinwoodward) [inline LICENSE-CODE:3]:
```suggestion
Copyright (c) GitHub
```

**Review by martinwoodward — CHANGES_REQUESTED** [review]
Update as the entity is just GitHub. Removing the year as we capture and show the date/time in the metadata of the files and discussion threads in this repo so save you updating the license file every year.

**Review by Vikus97 — COMMENTED** [review]
add-license-1

File: LICENSE-CODE (hunk 1 above)
Comment (koolbrezz80) [reply LICENSE-CODE:3]: I'm the onwer of accounts and y'all help someone steal from me

**Comment by koolbrezz80** [comment]
I been recording all of this since yesterday

**Comment by koolbrezz80** [comment]
I'm the onwer of the accounts

**Comment by Valja64** [comment]
> I been recording all of this since yesterday

**Comment by martinwoodward** [comment]
LGTM

**Review by martinwoodward — APPROVED** [review]

**Review by martinwoodward — APPROVED** [review]

**Comment by ettaboyle** [comment]
Thank you for completing this work @ossanna16 and for the additional support @martinwoodward 
This is only appearing at the repo level: https://github.com/community/community
- see screenshot: 
Screen Shot 2022-09-13 at 7 27 14 PM

We need this to be elevated to the org level discussions: https://github.com/orgs/community/discussions -- Do you know how to solve this? @michellemerrill and I tried to solve a similar issue closer to launch in July so tagging her + @bailey & @jricciardi  for visibility
Screen Shot 2022-09-13 at 7 30 21 PM

**Comment by martinwoodward** [comment]
Yeah, that would be a new feature request for the Discussions team at GitHub to present license info on the org main discussions page if present using the same logic as on the code view page.

The fact we have the license there should keep most lawyers happy but def be good to make it more discoverable for people who have visited the discussions page for a project (esp Org discussions)

**Comment by ettaboyle** [comment]
@martinwoodward super helpful! Thanks for the quick response @ossanna16 can you please add this to our [Discussions Feature Wishlist & Pain Points](https://github.com/github/community-engagement/discussions/2585)? It's not urgent, but should be tracked. You can include a link to this thread for context.

**Comment by ossanna16** [comment]
@ettaboyle Done ✅

**Review by Coinsintegrity — COMMENTED** [review]
> LICENSE-CODE``

**Comment by Coinsintegrity** [comment]
> > CÓDIGO DE LICENCIA``

**Comment by 0xw5555** [comment]
🔥

**Pull request merged**